src/components/accordion/CdrAccordion.vue

```diff
@@ -1,42 +1,187 @@
+<script setup>
+import {
+  useCssModule, computed, watch, onMounted, ref, inject,
+} from 'vue';
+import IconCaretDown from '../icon/comps/caret-down';
+import { modifyClassName } from '../../utils/buildClass';
+import mapClasses from '../../utils/mapClasses';
+
+
+const props = defineProps({
+  /**
+   * The unique id of an accordion.
```

Comment:
I think what we should be calling these are [disclosures ](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details) rather than accordion which to me is the grouping of these elements